packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,140 @@
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" provisioning_status_enum DEFAULT 'PENDING'::provisioning_status_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        "applicationId" uuid,
+        CONSTRAINT "fk_environment_application" FOREIGN KEY ("applicationId") REFERENCES application(id)
+    );
+    
+    ALTER TABLE environment OWNER TO atat_api_admin;
+    
+    CREATE TABLE portfolio (
+        id uuid DEFAULT uuid_generate_v4() CONSTRAINT "pk_portfolio" PRIMARY KEY,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" provisioning_status_enum DEFAULT 'PENDING'::provisioning_status_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        description character varying(300),
+        owner character varying NOT NULL,
+        csp csp_enum NOT NULL,
+        "dodComponents" character varying[] NOT NULL,
+        "portfolioManagers" character varying[] NOT NULL
+    );
+    
+    ALTER TABLE portfolio OWNER TO atat_api_admin;
+    
+    CREATE TABLE task_order (
+        id uuid DEFAULT uuid_generate_v4() CONSTRAINT "pk_task_order" PRIMARY KEY,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "taskOrderNumber" character varying(17) NOT NULL,
+        "fileId" uuid NOT NULL,
+        "fileName" character varying(256) NOT NULL,
+        "fileSize" integer,
+        "fileScanStatus" file_scan_status_enum DEFAULT 'PENDING'::file_scan_status_enum NOT NULL,
+        "portfolioId" uuid,
+        CONSTRAINT "fk_task_order_portfolio" FOREIGN KEY ("portfolioId") REFERENCES portfolio(id)
+    );
+    
+    ALTER TABLE task_order OWNER TO atat_api_admin;
+    
+    CREATE TABLE typeorm_metadata (
+        type character varying NOT NULL,
+        database character varying,
+        schema character varying,
+        "table" character varying,
+        name character varying,
+        value text
+    );
+    
+    ALTER TABLE typeorm_metadata OWNER TO atat_api_admin;
+    `;
```

Comment:
Could this live in it's own file? And we just import it to this file?

Reply:
It could.

Reply:
Implemented in e09df7c71ef60c48406483fd248e3188346dc508